e2e/utils/graphql.ts

```diff
@@ -62,21 +62,31 @@ const createGaloyServerClient = (config: Config) => (authToken: string) => {
   })
 }
 
-const getRandomToken = (arr: string[]): string => {
-  const randomIndex = Math.floor(Math.random() * arr.length)
-  console.log("Choosing token at random index: ", randomIndex)
-  return arr[randomIndex]
+const getRandomPhoneNumber = (): string => {
+  const randomDigits = Math.floor(Math.random() * 40 + 60) // Generates a number between 60 and 99
+    .toString()
+  return `+503650555${randomDigits}`
```

Comment:
why not getting a number from a much larger pool such that there would statistically never be a potential existing account?

Reply:
That's a good idea, we can fund that account initially using the test user and replenish test user's funds once in a while